src/routes/+layout.server.ts

```diff
@@ -0,0 +1,31 @@
+import { locales, loadTranslations, translations, defaultLocale } from "$lib/translations";
+import type { ServerLoad } from "@sveltejs/kit";
+import type { Translations } from "../../node_modules/sveltekit-i18n/node_modules/@sveltekit-i18n/base";
```

Comment:
I've made an issue regarding this:
https://github.com/sveltekit-i18n/lib/issues/181